test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -269,28 +231,6 @@ private void doWait(long millis) {
         }
     }
 
-    private int recover() {
```

Comment:
The recover is triggered by the WildFly startup

Reply:
What if we test on a different container?

Reply:
That's a good point @mmusgrov ! Would you rather keep the recover method for different container testing?